.ci/setup_env_fedora.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+#
+# Copyright (c) 2017-2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set -e
+
+cidir=$(dirname "$0")
+source /etc/os-release
+source "${cidir}/lib.sh"
+get_cc_versions
+
+arch=$(arch)
+
+if [ "$arch" = x86_64 ]; then
```

Comment:
This is the same check for ubuntu , I wonder if you could move it to `setup.sh`  
https://github.com/kata-containers/tests/blob/master/.ci/setup_env_ubuntu.sh#L17